### ISSUE TITLE
[insights-agent] Trivy namespace allow list now mandatory?

### ISSUE THREAD

**Issue by bkaidbb** [reactiveops/charts#1152]
### What happened?

The recent PR #1142 to Trivy to allow namespace allow list now makes that allow list mandatory.  If namespaces to allow aren't specified, no namespaces are scanned:

```
level=info msg="Found 0 images in cluster"
```

### What did you expect to happen?

I'd expect if no allow list values were defined, it would continue to scan all of them like it used to:

```
level=info msg="Found 38 images in cluster"
```

### How can we reproduce this?

values.yaml:

```
trivy:
  enabled: true
```

### Version

2.16.1

### Search

- [X] I did search for other open and closed issues before opening this.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by rbren** [comment]
Thanks for the report! This was a bug in the trivy plugin. It's fixed in plugin version `0.26.1` which gets pulled into the chart automatically

**Comment by bkaidbb** [comment]
@rbren Can you check https://github.com/FairwindsOps/charts/pull/1162 which bumps trivy plugin? The new version doesn't get pulled in automatically.  I have confirmed 0.26.1 does fix the issue though.
